src/Compilers/VisualBasic/Portable/VisualBasicCompilationOptions.vb

```diff
@@ -253,9 +253,16 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
         Friend Overrides Function GetImports() As ImmutableArray(Of String)
             ' TODO: implement (only called from VBI) https://github.com/dotnet/roslyn/issues/5854
+            Return ImmutableArray.CreateRange(GetImportList())
```

Comment:
It's probably not important here, given that this code doesn't run very often, but I've been encouraged to use ArrayBuilder for creating immutable arrays--something like:

Dim names As New ArrayBuilder(Of String)(GlobalImports.Length)  
names.AddRange(GlobalImports.Select(Function(go) go.Name))  
Return names.ToImmutableAndFree()

Someone with a deeper understanding than I of the performance implications of array creation could provide better guidance.